lib/puppet/pops/validation/validator_factory_4_0.rb

```diff
@@ -28,7 +28,6 @@ def severity_producer
     p[Issues::FUTURE_RESERVED_WORD]          = :deprecation
 
     p[Issues::DUPLICATE_KEY]                 = Puppet[:strict] == :off ? :ignore : Puppet[:strict]
-    p[Issues::DUPLICATE_DEFAULT]             = Puppet[:strict] == :off ? :ignore : Puppet[:strict]
```

Comment:
I think we don't want commit c2d14d8359ac51d65e881f2e6b33b67a0ed4ccd4 as it makes it hard error always in 4.10.x.